### ISSUE TITLE
Temperature alert in Aqara mini switch WXKG11LM

### ISSUE THREAD

**Issue by PTM26679** [veonua/SmartThingsEdge-Xiaomi#8]
First of all, THANK YOU for this driver! It's the only one that fully works (single click/hold/double click)

Not really a "real" issue, more of a question:
I installed the Aqara WXKG11LM mini switch and below the 3 button states, I see a "Temperature alert" but doesn't work. What's its purpose?
![photo_2021-12-26_02-28-41](https://user-images.githubusercontent.com/95320710/147396057-cbdd31d5-4e44-439e-8ffe-e25549cf7a98.jpg)

**Comment by veonua** [comment]
every zigbee chip has internal inaccurate  temperature sensor +-4*C.
so it is good to detect extreme temperatures only , now the alert will
trigger when device out of the recommended temperature range -15*..50*

tldr; probably it can detect overheating and 🔥

**Comment by PTM26679** [comment]
Wow! 
I had no idea about that!
Thank you!
